server/events/comment_parser.go

```diff
@@ -404,6 +407,8 @@ Commands:
 {{- end }}
   unlock   Removes all atlantis locks and discards all plans for this PR.
            To unlock a specific plan you can use the Atlantis UI.
+  approve_policies
+           Approves all current policy checking violations for the PR.
```

Comment:
Not sure if `violations` is the best term here, there isn't much in the docs. Perhaps `errors` or `exceptions` would be better?

Reply:
Changed to `failure` to better align with terminology in use.